workspace/commands/merge.py

```diff
@@ -151,23 +152,23 @@ def run(self):
                 'Please specify either a branch to merge from or --downstreams to merge to all downstream branches')
             sys.exit(1)
 
-    def merge_commits(self, branch_name, unmerged_commits_string, whitelist_for_ours_strategy=None):
+    def merge_commits(self, branch_name, unmerged_commits_string, skip_commits=None):
         """
-        Function to merge the unmerged commits. If  whitelist_for_ours_strategy is empty, it will merge using the heads
+        Function to merge the unmerged commits. If  skip_commits is empty, it will merge using the heads
         of the source and destination(current) branch.
 
-        If whitelist_for_ours_strategy is present, then we will inspect the list of commits for a match with the text
-        from whitelist_for_ours_strategy. If match is not found the commit will be merged with whatever strategy passed
+        If skip_commits is present, then we will inspect the list of commits for a match with the text
+        from skip_commits. If match is not found the commit will be merged with whatever strategy passed
         to the class else if a match is found it will merge that specific commit with `ours` strategy.
 
         :param branch_name: Name of the source branch
         :param unmerged_commits_string: unmerged_commits separated with a \n
-        :param whitelist_for_ours_strategy: list of the commit texts to look for.
+        :param skip_commits: list of the commit texts to look for.
```

Comment:
Can we use the same param description as above for skip_commits?

Reply:
sure.